pkg/medusa/reconcile.go

```diff
@@ -88,6 +88,11 @@ func CreateMedusaIni(kc *k8ss.K8ssandraCluster) string {
     {{- else }}
     secure = True
     {{- end }}
+    {{- if not .Spec.Medusa.StorageProperties.SslVerify }}
+    ssl_verify = False
+	{{- else }}
+	ssl_verify = True
```

Comment:
**Issue:** I think the indentation is wrong here.

Reply:
Great catch, it looked all good in the IDE.